README.md

```diff
@@ -20,52 +20,38 @@ The following environment variables are used:
 
 ## Developement Setup
 
+Before working on this repo, you should already have [set up a bot account](https://discordjs.guide/preparations/setting-up-a-bot-application.html#creating-your-bot) and [added it to your development server](https://discordjs.guide/preparations/adding-your-bot-to-servers.html). In order to mimic the Codewars Discord server in your development server, you may also wish to add appropriate roles such as @admin and @mods, as well as common channels such as #help-solve.
+
 > NOTE: Please discuss with us first before adding new features to avoid wasting your time.
 
 > TODO Expand
 
-### Install dependencies and build
-
-```bash
-npm install
-```
-
-### Start bot with your token
-
-```bash
-BOT_TOKEN=.... npm start
-```
-
-### Making changes
+After forking this repo and cloning your fork to your local development environment:
 
-Let TypeScript compile on change:
+1. Change directory to the root of this repo: `$ cd /path/to/your/discord-bot`
+1. Install dependencies: `$ npm install`
+1. Compile the TypeScript source files: `$ npm run build`
+1. Run your bot with [your token](https://discordjs.guide/preparations/setting-up-a-bot-application.html#your-token): `$ BOT_TOKEN=... npm start`
 
-```bash
-npm run build:watch
```

Comment:
This should be the most efficient way to work. Keep this running while you make changes and just restart the bot.